src/hooks/useSnackbar/useSnackbar.tsx

```diff
@@ -0,0 +1,100 @@
+import Snackbar from '@/shared/components/Snackbar/Snackbar'
+import { transitions } from '@/shared/theme'
+import { css } from '@emotion/react'
+import React, { createContext, useContext, useEffect, useState } from 'react'
+import { CSSTransition } from 'react-transition-group'
+
+export type DisplaySnackbarArgs = {
+  time?: number
+  variant?: 'success' | 'error' | 'info'
+  buttonText?: string
+  message: string
+}
+
+const snackbarTransitions = css`
```

Comment:
maybe it's better to move transition style to `Snackbar.style` to keep hook file cleaner. Or is there a reason why it's here?